xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-plugins/src/main/resources/xwiki-macro/plugin.js

```diff
@@ -448,7 +448,17 @@
           var inlineEnforcer = editor.editable().findOne('span#xwiki-macro-inline-enforcer');
           if (inlineEnforcer) {
             // Place the caret after the inserted inline macro in order to allow the user to continue typing.
-            editor.getSelection().selectElement(inlineEnforcer);
+            // We proceed by inserting and non-breakable space after the inline enforcer, before selecting its content.
+            // It is required to be cross-browser compatible. Without these operations, the caret was not visible in
+            // Chrome after the inline macro insertion.
+            var space = new CKEDITOR.dom.text('\u00A0');
+            space.insertAfter(inlineEnforcer);
+            var range = editor.createRange();
+            range.selectNodeContents(space);
+            // TEST with not selected space.
+            range.setStartAfter(space, range.endOffset);
```

Comment:
This is optional and is only needed if we decide that's it better not to have the non-breakable space not selected after insert.